petclinic-frontend/src/features/inventories/InventoriesListTable.tsx

```diff
@@ -94,6 +97,39 @@ export default function InventoriesListTable(): JSX.Element {
     }
   };
 
+  useEffect(() => {
+    if (inventoryList.length > 0) {
+      inventoryList.forEach(inventory => {
+        fetchProductQuantity(inventory.inventoryId);
+      });
+    }
+  }, [inventoryList]);
```

Comment:
I think binding the product quantity of an inventory to the event where the inventoryList would be changed is a good idea. Later, we could also make it so that when a product within an inventory is added or deleted, that the action would notify its inventory that it's amount of products has changed.

Reply:
Good idea, thanks for the feedback!